build.gradle

```diff
@@ -81,95 +79,23 @@ dependencies {
     testImplementation "com.google.truth:truth:1.0.1"
 }
 
-// Maven publishing settings (nexus-maven-plugins)
-modifyPom {
-    project {
-        name "Bugsnag Android Gradle Plugin"
-        description "Gradle plugin to automatically upload ProGuard mapping files to Bugsnag."
-        url "https://github.com/bugsnag/bugsnag-android-gradle-plugin"
-
-        scm {
-            url "https://github.com/bugsnag/bugsnag-android-gradle-plugin"
-            connection "scm:git:git://github.com/bugsnag/bugsnag-android-gradle-plugin.git"
-            developerConnection "scm:git:ssh://git@github.com/bugsnag/bugsnag-android-gradle-plugin.git"
-        }
-
-        licenses {
-            license {
-                name "MIT"
-                url "http://opensource.org/licenses/MIT"
-                distribution "repo"
-            }
-        }
-
-        organization {
-            name "Bugsnag"
-            url "https://bugsnag.com"
-        }
-
-        developers {
-            developer {
-                id "loopj"
-                name "James Smith"
-                email "james@bugsnag.com"
-            }
-        }
-    }
-}
-
 // Gradle plugin publishing settings (plugins.gradle.com)
 pluginBundle {
-    website = "https://github.com/bugsnag/bugsnag-android-gradle-plugin"
-    vcsUrl = "https://github.com/bugsnag/bugsnag-android-gradle-plugin.git"
+    website = POM_URL
+    vcsUrl = POM_SCM_CONNECTION
 
     plugins {
         androidGradlePlugin {
             id = "com.bugsnag.android.gradle"
-            displayName = "Bugsnag Android Gradle Plugin"
-            description = "Gradle plugin to automatically upload ProGuard mapping files to Bugsnag."
+            displayName = POM_NAME
+            description = POM_DESCRIPTION
             tags = ["bugsnag", "proguard", "android", "upload"]
         }
     }
 
     mavenCoordinates {
-        groupId = "com.bugsnag"
-        artifactId = "bugsnag-android-gradle-plugin"
-    }
-}
-
-install {
-    version = project.hasProperty("localVersion") ? project.getProperty("localVersion") : project.version
-}
-
-publishing {
-    publications {
-        Publication(MavenPublication) {
-            artifact jar
-            groupId "com.bugsnag"
-            artifactId "bugsnag-android-gradle-plugin"
-            version project.version
-        }
-    }
-}
-
-// Bintray upload
-bintray {
```

Comment:
One of our requirements is publishing the plugin to mavenCentral, plugins.gradle, and bintray. Is there an equivalent way of publishing to bintray when using the maven publish plugin?

Reply:
JCenter will source maven central to mirror anything it doesn't have. I think the plug-in could handle bintray uploads but would need to check. Does the mirroring not cover the bintray need?

Reply:
We've discussed this internally and are happy to drop publishing to bintray. The only reason it was added was that several years ago we had issues with mavenCentral's reliability, which haven't been experienced recently. We'll probably hold off merging this until the next major of the bugsnag gradle plugin is released to reduce the scope of changes - hopefully this should be quite soon.